web-ui/src/components/menu/__snapshots__/Menu.test.jsx.snap

```diff
@@ -33,8 +33,8 @@ exports[`<Menu /> > renders correctly 1`] = `
           />
         </button>
       </div>
-      <div
-        aria-haspopup="true"
+      <a
+        href="/profile/undefined"
```

Comment:
I wonder if the reason this contains "undefined" is that in the test we don't have a user id.

Reply:
In `Menu.test.jsx` currently I see us taking snapshots, but I don't see us making any assertions yet. In this case, the `${id}` is set at runtime based on the user profile. If we want to test various render conditions, and just _that_ it renders, I think we have room to beef up our coverage.

Reply:
I added a test to verify that the Link around the Avatar uses the id value.